web/src/app/schedules/temp-sched/TempSchedAddNewShift.tsx

```diff
@@ -184,7 +184,7 @@ export default function TempSchedAddNewShift({
                 name='shift-start'
                 fieldName='start'
                 min={value.start}
-                max={value.end}
+                max={DateTime.fromISO(value.end).plus({ year: 1 }).toISO()}
```

Comment:
Do we have `zone` available for these calls?

```suggestion
                max={DateTime.fromISO(value.end, { zone }).plus({ year: 1 }).toISO()}
```